pkg/apis/cloud/v1beta1/backend.go

```diff
@@ -38,6 +38,15 @@ type Backend struct {
 
 	// VolumeClaim is an optional persistent volume claim to use for persistence
 	VolumeClaim *corev1.PersistentVolumeClaim `json:"volumeClaim,omitempty"`
+
+	// LivenessProbe backend livenessProbe configuration
+	LivenessProbe *corev1.Probe `json:"livenessProbe,omitempty"`
```

Comment:
We can also avoid adding these new fields and handle it in each controller but I think make it configurable gives more flexibility